Scripts/VerifyNoBuildSettings.swift

```diff
@@ -29,7 +29,10 @@ let ignoreSettings = [
   "PROVISIONING_PROFILE_SPECIFIER",
   "INFOPLIST_FILE",
   "TEST_TARGET_NAME",
-  "IBSC_MODULE"
+  "IBSC_MODULE",
+  "SDKROOT",
+  "EFFECTIVE_PLATFORM_NAME",
+  "PRODUCT_NAME"
```

Comment:
Do not touch this script. Please read the error message and follow instruction instead